src/CarbonAware.WebApi/src/Properties/launchSettings.json

```diff
@@ -13,11 +13,10 @@
       "commandName": "Project",
```

Comment:
Not entirely sure why my settings ended up changing - want me to revert this?

Reply:
@dylHall - yep, revert, I imagine this is something coming from the IDE or an extension trying to be smart.